src/jit/gentree.cpp

```diff
@@ -15399,13 +15312,9 @@ bool GenTree::IsPhiDefn()
     return res;
 }
 
-bool GenTree::IsPhiDefnStmt()
+bool GenTreeStmt::IsPhiDefnStmt()
```

Comment:
Maybe just remove this? There aren't many uses and it's just `gtStmtExpr->IsPhiDefn()`.

Reply:
I will remember it for the next change if you don't mind.